src/components/Breadcrumbs/index.css

```diff
@@ -0,0 +1,16 @@
+.Breadcrumbs-items {
+  display: flex;
```

Comment:
We need to ensure the layout breaks to multiple rows when on smaller viewports:

![image](https://user-images.githubusercontent.com/893208/38317876-f5f7acf0-3825-11e8-9efc-531a4418ef41.png)

Reply:
👍